dials/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
I'm glad we're done with that era :-)
Seriously, it's all boiler plate code and templates. I don't see what we can test here.